securedrop/journalist_app/__init__.py

```diff
@@ -144,5 +138,35 @@ def setup_g():
                            url_prefix='/account')
     app.register_blueprint(admin.make_blueprint(config), url_prefix='/admin')
     app.register_blueprint(col.make_blueprint(config), url_prefix='/col')
+    api_blueprint = api.make_blueprint(config)
+    app.register_blueprint(api_blueprint, url_prefix='/api/v1')
```

Comment:
One of the nice reasons to implement the API as a blueprint registered the main journalist app is that it's extremely simple to deploy and does not add complexity to an already quite complex ops story. Is there a major advantage to keeping the two separate that would justify the Apache config changes?

Reply:
From an ops perspective (in the context of SD), I think this is the correct thing to do. I just generally like to keep the apps separate when I deploy things because it prevents surprise behavior between different components IMO. Probably should have clarified my initial comment to better say "I don't like this particularly but also what you did is the only sane thing to do."